src/sgp/Logger.h

```diff
@@ -8,19 +8,22 @@
 void LogMessage(bool isAssert, LogLevel level, const char* file, const ST::string& str);
 void LogMessage(bool isAssert, LogLevel level, const char *file, const char *format, ...);
 
+/** Get filename relative to src directory */
+#define __FILENAME__ (__FILE__ + SOURCE_PATH_SIZE)
```

Comment:
Mac worker really doesn't like this, so the build logs are full of warnings (sextupling their size):
```
2020-07-12T10:38:01.4168070Z /Users/runner/work/ja2-stracciatella/ja2-stracciatella/src/game/SaveLoadGame.cc:1304:3: warning: adding 'int' to a string does not append to the string [-Wstring-plus-int]
2020-07-12T10:38:01.4169070Z                 Assert(s->ubID == i);
2020-07-12T10:38:01.4169370Z                 ^~~~~~~~~~~~~~~~~~~~
2020-07-12T10:38:01.4170570Z /Users/runner/work/ja2-stracciatella/ja2-stracciatella/src/sgp/Debug.h:10:41: note: expanded from macro 'Assert'
2020-07-12T10:38:01.4170880Z #define Assert(a)       (a) ? (void)0 : SLOGA("Assertion failed in %s, line %d", __FILE__, __LINE__)
2020-07-12T10:38:01.4171160Z                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2020-07-12T10:38:01.4172240Z /Users/runner/work/ja2-stracciatella/ja2-stracciatella/src/sgp/Logger.h:27:62: note: expanded from macro 'SLOGA'
2020-07-12T10:38:01.4172580Z #define SLOGA(FORMAT, ...) LogMessage(true, LogLevel::Error, __FILENAME__, FORMAT, ##__VA_ARGS__)
2020-07-12T10:38:01.4172870Z                                                              ^~~~~~~~~~~~
2020-07-12T10:38:01.4173830Z /Users/runner/work/ja2-stracciatella/ja2-stracciatella/src/sgp/Logger.h:12:32: note: expanded from macro '__FILENAME__'
2020-07-12T10:38:01.4174140Z #define __FILENAME__ (__FILE__ + SOURCE_PATH_SIZE)
2020-07-12T10:38:01.4174390Z                       ~~~~~~~~~^~~~~~~~~~~~~~~~~~
```

Reply:
Fixed via #1246